src/components/search-input/search-input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./search-input.scss";
+import { FaSearch } from "react-icons/fa";
+
+export const SearchInput = ({
+  hasIcon = true,
+  onValueChange,
+  placeholder = "cauta",
+  value
+}) => {
+  const [currentValue, setCurrentValue] = useState("");
+  return (
+    <div className={`__search-input-container ${hasIcon ? "__has_icon" : ""}`}>
+      <input
+        placeholder={placeholder}
+        value={value || currentValue || ""}
```

Comment:
Once the changes above are added, we can have just `currentValue`
```suggestion
        value={currentValue}
```